odsApi_versioned_docs/version-7.2/platform-dev-guide/configuration/readme.md

```diff
@@ -11,7 +11,7 @@ instance of the Ed-Fi ODS / API. By customizing the `appsettings.json` you
 should be able to modify the API to suit your needs. `appsettings.json` includes
 settings for database connection strings and the settings that enable various
 configurable features. For more detail on the configuration files and the
-specific values, see the [Configuration Details](./configuration-details.md)
+specific values, see the [Configuration Details](./configuration-details.mdx)
```

Comment:
can you revert this change. As 7.2 space still has the md file, build is reporting broken link

Reply:
Done.